src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+    protected String by;
+
+    public Deadline(String description, String by){
+        super(description);
+        this.by = by;
+    }
+
+    public String toString(){
+        return "[D]" + super.toString() +" (by: " + by +")";
+    }
+
+}
```

Comment:
I like how you add a space after every countructor/objects in this class, this makes code-reading easier for audiences like me! :)